docs/cne-install.md

```diff
@@ -0,0 +1,177 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See [Install the System Admin Toolkit Product Stream](install.md) for installation instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both automated and logged to help you save time. The CNE installer can be used to upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000) for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
+- cray-product-catalog is running.
+- There must be at least 2 gigabytes of free space on the manager NCN on which the
+  procedure is run.
+
+### Notes on the Procedures
+
+- Ellipses (`...`) in shell output indicate omitted lines.
+- In the examples below, replace `x.y.z` with the version of the SAT product stream
+  being upgraded.
+- 'manager' and 'master' are used interchangeably in the steps below.
+
+### Pre-Upgrade Procedure
+
+1.  Start a typescript and set the shell prompt.
+
+    The typescript will record the commands and the output from this upgrade.
+    The prompt is set to include the date and time.
+
+    ```screen
+    ncn-m001# script -af product-sat.$(date +%Y-%m-%d).txt
+    ncn-m001# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+    ```
```

Comment:
nit: per the lint rules we discussed last week, there should only be one space here. In doing this, could you please also dedent the continuation lines of this step by one space to match, i.e.:
~~~suggestion
1. Start a typescript and set the shell prompt.

   The typescript will record the commands and the output from this upgrade.
   The prompt is set to include the date and time.

   ```screen
   ncn-m001# script -af product-sat.$(date +%Y-%m-%d).txt
   ncn-m001# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
   ```
~~~